recipes/match-series/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "match-series" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/berkels/{{ name }}/archive/master.tar.gz
```

Comment:
Can you try and get upstream to tag a version?

Reply:
I create a request upstream https://github.com/berkels/match-series/issues/2